src/components/Auth/components/Profile/Profile.tsx

```diff
@@ -94,10 +95,34 @@ export default function Profile() {
       setHandleUpdateError(error.message);
       return;
     }
-    setSuccessAlert(true);
     setProfile(data);
+    setSuccessAlert(true);
   };
 
+  useEffect(() => {
+    if (getProfileError) {
+      setAlertMessage({ type: "error", message: `${getProfileError}` });
```

Comment:
Aquí puedes pasar la variable sin las comillas
```suggestion
      setAlertMessage({ type: "error", message: getProfileError});
```